mobify/shade
# Updating grunt tasks. Inlining CSS to remove stylesheet dependency.

Inlining CSS to remove the need to include a stylesheet with the plugin.

Status: **Ready to merge**

Reviewers: @tedtate @kpeatt 
## Changes
- inlined CSS
- renamed grunt build-dist to just build
- removed unused modules
- updated mobify-code-style
## How to test-drive this PR
- checkout this branch
- `grunt test`
- point to this branch in a pinny, and ensure it works
